integrations/extensions/docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md

```diff
@@ -0,0 +1,250 @@
+# How to set up Elasticsearch from IBM Cloud and integrate it with Watson Assistant
+This is a documentation about how to set up Elasticsearch from IBM Cloud and create Watson Assistant search extension using Elasticsearch index.
+
+## Table of contents:
+* [Step 1: Provision an Elasticsearch instance on IBM Cloud](#step-1-provision-an-elasticsearch-instance-on-ibm-cloud)
+* [Step 2: Set up Kibana to connect to Elasticsearch](#step-2-set-up-kibana-to-connect-to-elasticsearch)
+* [Step 3: Create an Elasticsearch index (keyword-search)](#step-3-create-an-elasticsearch-index-keyword-search)
+* [Step 4: Set up Watson Assistant search extension using Elasticsearch index](#step-4-set-up-watson-assistant-search-extension-using-elasticsearch-index)
+* [Step 5: Enable semantic search with ELSER](#step-5-enable-semantic-search-with-elser)
+
+
+## Step 1: Provision an Elasticsearch instance on IBM Cloud
+* Create an [IBM Cloud account](https://cloud.ibm.com/registration) if you don't have one.
+* Provision a Databases for Elasticsearch instance from the [IBM Cloud catalog](https://cloud.ibm.com/catalog/databases-for-elasticsearch).  
+  **A platinum plan with at least 4GB RAM is required in order to use the advanced ML features,
+  such as [Elastic Learned Sparse EncodeR (ELSER)](https://www.elastic.co/guide/en/machine-learning/8.10/ml-nlp-elser.html)**
+* Create a service credentials from the left-side menu and find the `hostname`, `port`, `username` and `password`.
+  The credentials will be used to connect to Kibana and Watson Assistant at next steps. You can use admin userid and password as well.
+  Please refer to [this doc](https://cloud.ibm.com/docs/databases-for-elasticsearch?topic=databases-for-elasticsearch-user-management&interface=ui#user-management-elasticsearch-ibm-superuser) to learn more about different user roles.
+
+
+## Step 2: Set up Kibana to connect to Elasticsearch
+* Install [Docker](https://www.docker.com/) so that you can pull the Kibana container image later.
```

Comment:
This is saying to install Docker.  Is this a reference to Docker Desktop or something else (sorry, I know very little about container management software)?

My understanding is that this is not allowed for IBM employees as per https://w3.ibm.com/w3publisher/third-party-software/strategic-software-migration/software-development-tools .  Is it possible to do these steps using an alternative tool such as the ones listed on that page?  There are more details at https://w3.ibm.com/w3publisher/docker-desktop .

Reply:
Good point! I added a comment there to install `Colima` if we don't want to use `Docker Desktop`. All the other steps stay the same.